src/redux-query-sync.js

```diff
@@ -71,7 +71,7 @@ function ReduxQuerySync({
         // For each parameter value that changed, call the corresponding action.
         Object.keys(queryValues).forEach(param => {
             const value = queryValues[param]
-            if (value !== lastQueryValues[param]) {
+            if (value && value !== lastQueryValues[param]) {
```

Comment:
Right, my bad, but still I added ` && value !== undefined`.
In case `defaultValue` is undefined, I see a bug related to the possibility to move back in history, before the first page of the app.
I wonder if that bug can be solved in another way..